File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/message/SeatsReservedMessage.cs
using System;
using System.Collections.Generic;
using ENode.Infrastructure;

namespace ConferenceManagement.Messages
{
    public class SeatsReservedMessage : ApplicationMessage
    {
        public Guid ConferenceId { get; set; }
        public Guid ReservationId { get; set; }
        public IEnumerable<SeatReservationItem> ReservationItems { get; set; }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/message/SeatsReservedMessage.java
package com.microsoft.conference.common.management.message;

import org.enodeframework.messaging.ApplicationMessage;

import java.util.List;

public class SeatsReservedMessage extends ApplicationMessage {
    public String conferenceId;
    public String reservationId;
    public List<SeatReservationItem> reservationItems;
}
